app/views/replies/preview.haml

```diff
@@ -1,3 +1,17 @@
+- if @unseen_replies.present?
+  .post-header
+    .padding-15
+      Unseen Replies
+      = link_to post_path(@post, page: 'unread', anchor: 'unread'), target: '_blank' do
+        .view-button Unread (new tab) &raquo;
+      = link_to post_path(@post, page: 'unread', anchor: 'unread') do
+        .view-button.float-left Unread &raquo;
```

Comment:
I don't think two buttons are necessary here; UI usually provides either a text link (same tab) with an associated "new window" glyph (new tab), or it just sticks to one or the other. It doesn't really seem useful to have both, and I'd stick to just the `target: '_blank'` one.